server/src/matchRecipes.js

```diff
@@ -50,6 +58,14 @@ type MatchedRecipeResult = {
   xid: string,
   name: string,
   rating: number,
+  calories: number,
+  total_fat: number,
```

Comment:
Please use camelCase for JavaScript object properties

Reply:
Sorry about the snake_case, I will fix it before merging this commit.